---
title: "A benchmark for RNA-seq deconvolution analysis under dynamic testing environments"
authors: ["Haijing Jin", "Zhandong Liu"]
year: 2021
venue: "Genome Biology"
type: "paper"
cover: "../../assets/paper-vision.jpg"
links:
  pdf: "Available Version (via Google Scholar):/Users/hyun-hwanjeong/Zotero/storage/MWLBA9A8/s13059-021-02290-6.html:text/html;Full Text:/Users/hyun-hwanjeong/Zotero/storage/Y6NTQ8WJ/Jin and Liu - 2021 - A benchmark for RNA-seq deconvolution analysis und.pdf:application/pdf"
  code: ""
  website: ""
  demo: ""
  slides: ""
  video: ""
doi: "10.1186/s13059-021-02290-6"
description: "Abstract
            
              Background
              Deconvolution analyses have been widely used to track compositional alterations of cell types in gene expression data. Although a large number of novel methods have been developed, due to a lack of understanding of the effects of modeling assumptions and tuning parameters, it is challenging for researchers to select an optimal deconvolution method suitable for the targeted biological conditions.
            
            
              Results
              To systematically reveal the pitfalls and challenges of deconvolution analyses, we investigate the impact of several technical and biological factors including simulation model, quantification unit, component number, weight matrix, and unknown content by constructing three benchmarking frameworks. These frameworks cover comparative analysis of 11 popular deconvolution methods under 1766 conditions.
            
            
              Conclusions
              We provide new insights to researchers for future application, standardization, and development of deconvolution tools on RNA-seq data."
featured: false
---
Abstract
            
              Background
              Deconvolution analyses have been widely used to track compositional alterations of cell types in gene expression data. Although a large number of novel methods have been developed, due to a lack of understanding of the effects of modeling assumptions and tuning parameters, it is challenging for researchers to select an optimal deconvolution method suitable for the targeted biological conditions.
            
            
              Results
              To systematically reveal the pitfalls and challenges of deconvolution analyses, we investigate the impact of several technical and biological factors including simulation model, quantification unit, component number, weight matrix, and unknown content by constructing three benchmarking frameworks. These frameworks cover comparative analysis of 11 popular deconvolution methods under 1766 conditions.
            
            
              Conclusions
              We provide new insights to researchers for future application, standardization, and development of deconvolution tools on RNA-seq data.